Sources/Recoverable/RecoverableViewState.swift

```diff
@@ -8,22 +8,40 @@
 
 import UIKit
 
-struct RecoverableViewState {
+class RecoverableViewState {
```

Comment:
We can keep this as struct, I don't think that it makes sense to use `RecoverableViewState` as a base class, as we are creating a new state for each view type (`UILabel`, `UITextView`, ...) and not using a single view state with Polymorphism.

Reply:
Yes, you are right. I had a problem when on another file when they were structs (needed RecoverableTextViewState and imageState to be subclasses of RecoverableViewState), but I  fixed it in that end and have now changed them to structs instead in the latest commit.